<launch>
        <rosparam command="load" file="$(find catchrobo_driver)/config/hardware_config.yaml"/>
        <node name="catchrobo_driver" pkg="catchrobo_driver" type="catchrobo_driver.py" output="screen"/>
        <node name="joy_node" pkg="joy" type="joy_node" output="screen"/>
        <node name="gamepad_ctrl" pkg="catchrobo_driver" type="gamepad_ctrl.py" output="screen"/>
</launch>
